server/application/terminal.go

```diff
@@ -54,12 +58,17 @@ func (s *terminalHandler) getApplicationClusterRawConfig(ctx context.Context, a
 	return clst.RawRestConfig(), nil
 }
 
+// getSanitizedValue gets a sanitized value from url values
+func getSanitizedValue(q url.Values, key string) string {
+	return strings.Replace(q.Get(key), "\n", "", -1)
```

Comment:
Maybe we could use this regex and a 63-character limit check? https://stackoverflow.com/a/27805684/684776

Could then rename this function `isValidKubernetesObjectName` and add a check at the beginning for ServeHTTP.